### ISSUE TITLE
feat(datamodel): individual demographics attributes (like for Threat Actor Individuals)

### ISSUE THREAD

**Issue by ghost** [LuatixHQ/opencti#4732]
## Use case
To track individuals, like CEOs, Presidents, Ministers, ... it would be great to have the same demographic fields available like for Threat Actor individuals.

## Current Workaround
Note certain attributes like date of birth or job title in the description

## Proposed Solution
Implement the same demographic attributes like for threat actor individuals

**Comment by ghost** [comment]
@Jipegien can you please take a look at this feature request? Probably most of the implementation of the Threat Actor Individuals can be reused.

**Comment by Jipegien** [comment]
Yes I understand, but it won't be a priority at all for us. Feel free to propose a PR on this to speed up the delivery.
